buildSrc/src/main/java/org/opensearch/gradle/OpenSearchTestBasePlugin.java

```diff
@@ -109,7 +109,7 @@ public void execute(Task t) {
                         test.systemProperty("java.locale.providers", "SPI,JRE");
                     } else {
                         test.systemProperty("java.locale.providers", "SPI,COMPAT");
-                        test.jvmArgs("--illegal-access=warn");
+                        test.jvmArgs("--illegal-access=warn", "-Djava.security.manager=allow");
```

Comment:
@adnapibar thanks for the change, I believe we have to make it conditional based on JDK (the new `allow` setting was introduced in JDK-18 and onwards), otherwise JVM fails to start

Reply:
@reta thanks! for pointing that, I'm updating it.